db_management/questionnairedb/data/test/omi/patients.sql

```diff
@@ -2,4 +2,25 @@
 --
 -- SPDX-License-Identifier: AGPL-3.0-or-later
 
--- patients get synced automatically via the DB event that runs periodically
+-- Legacy (deleted) test patients are kept until all their remaining data in QuestionnaireDB is cleaned up
+-- Otherwise, data linked to certain patientIds will be orphaned
+REPLACE INTO `patient` (`ID`, `hospitalId`, `externalId`, `deleted`, `deletedBy`, `creationDate`, `createdBy`, `lastUpdated`, `updatedBy`) VALUES
+(1, -1, 51, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_MARGE_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(2, -1, 52, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_HOMER_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(3, -1, 53, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_BART_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(5, -1, 55, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_MONA_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(6, -1, 56, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_FRED_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(7, -1, 57, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_PEBBLES_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(8, -1, 58, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_WEDNESDAY_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+
+-- Laurie and Rory's patient rows are in reverse order in this database, as opposed to OpalDB.Patient
```

Comment:
What do you mean with this?

Reply:
Oh I see, as stated above, I suggest we add them all in which case we don't need to rely on the sync and can remove this comment.